Source/Models/FORMSection.m

```diff
@@ -33,6 +34,19 @@ - (instancetype)initWithDictionary:(NSDictionary *)dictionary
         [fields addObject:field];
     }];
 
+    if (_type == FORMSectionTypeDynamic) {
+        FORMField *field = [FORMField new];
+        field.position = @(fields.count);
+        field.section = self;
+        field.fieldID = [NSString stringWithFormat:@"%@.add", self.sectionID];
+        field.typeString = @"button";
+        field.type = FORMFieldTypeButton;
+        field.title = [NSString stringWithFormat:@"✚ %@", [self.sectionID uppercaseString]];
```

Comment:
I don't think it works that way =/ 

I was thinking on having a `title` attribute on the section/template, maybe that's the way!